apps/trading-e2e/src/integration/withdrawals.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Withdrawals
+
+  Scenario: Can prepare a withdrawal
+    Given I navigate to withdrawals page
```

Comment:
I think its a bit superfluous to navigate to the page each time and just lengthens tests needlessly. I'd prefer to have a Given which just starts on the right page. We can have a different navigation feature which can navigate to each page and make sure that all works

```suggestion
    Given I am on the withdrawals page
```

Reply:
update as part of     feat/104-withdraws